# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/LayerMetricsWrapper.h
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 gfx/layers/LayerMetricsWrapper.h
--- a/gfx/layers/LayerMetricsWrapper.h	Fri Mar 22 18:28:31 2019 +0000
+++ b/gfx/layers/LayerMetricsWrapper.h	Fri Mar 22 18:28:42 2019 +0000
@@ -331,6 +331,8 @@
     return Nothing();
   }
 
+  Maybe<wr::RenderRoot> GetReferentRenderRoot() const { return Nothing(); }
+
   Maybe<ParentLayerIntRect> GetClipRect() const {
     MOZ_ASSERT(IsValid());
 